flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/AlterSchemaConverter.java

```diff
@@ -206,6 +210,24 @@ public Operation convertAlterSchema(
                 dropColumn, tableChanges, schemaBuilder.build(), oldTable);
     }
 
+    /** Convert ALTER TABLE DROP DISTRIBUTION to generate an updated Schema. */
+    public Operation convertAlterSchema(
```

Comment:
the name is misleading as distribution is not part of the "schema". `convertAlterDistribution`?

Reply:
Done